clients/src/test/java/org/apache/kafka/common/network/SslTransportLayerTest.java

```diff
@@ -1250,7 +1322,7 @@ private NioEchoServer createEchoServer(SecurityProtocol securityProtocol) throws
         void run() throws IOException;
     }
 
-    private static class TestSslChannelBuilder extends SslChannelBuilder {
+    public static class TestSslChannelBuilder extends SslChannelBuilder {
```

Comment:
Do these have to be public or can they be package private?

Reply:
The correct scope is package private.
Fixed.